Tests/GRPCCoreTests/Call/TracingTestsUtils.swift

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2024, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Tracing
+import XCTest
+
+class TracingTestCase: XCTestCase {
+  override class func setUp() {
+    InstrumentationSystem.bootstrap(TestTracer())
```

Comment:
Does this blow up like logging/metrics?

Reply:
You mean if it fails when called more than once? If so, yes, and that's why I had to do it here, so that it's called just once.